File: lógica/JS/estrutura-condicional/menor-de-tres.js
// Fazer um programa para ler três números inteiros. Em seguida, mostrar qual o menor dentre os três
// números lidos. Em caso de empate, mostrar apenas uma vez.

let a, b, c, menor; 

a = 5;
b = 12;
c = 5;

if(a < b && a < c) {
    menor = a;
} else if(b < c) {
    menor = b;
} else {
    menor = c;
}

console.log('Menor = ' + menor);

File: lógica/Python/estrutura-condicional/menor-de-tres.py
""" Fazer um programa para ler três números inteiros. Em seguida, mostrar qual o menor dentre os três
números lidos. Em caso de empate, mostrar apenas uma vez. """

x = int(input("Primeiro inteiro: "))
y = int(input("Segundo inteiro: "))
z = int(input("Terceiro inteiro: "))
menor: int

if x < y and x < z:
    menor = x 
elif  y < z:
    menor = y
else:
    menor = z 

print(f"MENOR = {menor}")




